activation/licensed-users-count.md

```diff
@@ -30,6 +30,31 @@ The number of licensed Users within Syskit Point can be found by going to **Sett
 
 ![Browse Licensed Users by license](../.gitbook/assets/activation-licensed-users_browse-license.png)
 
+To access information on how many assigned licenses there are in your Microsoft 365, visit the admin center and take the following steps. 
+
+Under **Billing (1)**, select **Licenses (2)**. The list of **owned licenses (3)** will then be shown. 
+ * The **Available Licenses (4)** section is the number of licenses that are still available to be assigned to users. 
+ * The **Assigned Licenses (5)** section shows the number of assigned licenses compared to the available number. 
+
+![How to Check the Microsoft 365 Licenses](../.gitbook/assets/licensed-users-count_microsoft-licenses.png)
+
+Selecting each license allows you to **Assign** it to additional users or **Unassign** it from current ones. This can help with optimizing the cost of the SysKit Point subscription to suit your needs better.
+
+**To unassign users from a license** click on the license name to select it. This will open the **License details (1)** screen.  
+
+ * Once there, **select either the individual Users to unassign or the top box to select all Users (2)**.
+ * This will provide the option to click on **Unassign licenses (3)**; a pop-up will open requesting you click the **Unassign button to confirm**.
+
+The pricing for SysKit Point is **calculated based on each User that has a license assigned to them**. 
+For example, if one User has three licenses, the charge would still be for that one User, regardless of the number of licenses assigned to them. 
+
+![Unassign Microsoft 365 Licenses](../.gitbook/assets/licensed-users-count_microsoft-license-unassign.png)
```

Comment:
još jednom uzmi screenshot ove slike gdje je Toni Frankola. promijenio sam name usera da bude neki no-name umjesto Tonija

Reply:
baciti u grammarly ako još nisi. 
osim toga ovo što si nadopisala mi je super; samo još ove dvije izmjene i šaljemo mail Luki da vidi dal mu je ok